cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('User login functionality', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.wait(2000)
+  })
+
+  const openLoginForm = () => {
+    cy.get('#registerModal button.btn-close').click({ force: true })
+    cy.get('button[data-bs-target="#loginModal"]').eq(0).click({ force: true })
+  }
+
+  it('should switch to the login form when login button is clicked', () => {
+    openLoginForm()
+    cy.get('#loginForm').should('be.visible')
+  })
+
+  const loginUser = (email, password) => {
+    openLoginForm()
+    cy.wait(1000)
+    cy.get('#loginEmail').type(email)
+    cy.wait(1000)
```

Comment:
I don't think you need to wait here :)

Reply:
I had to wait to wait a bit, I think it was related to my computer where it was skipping the email field. So that wait made sure to insert the email into the email field